Libraries/oneTBB/tbb-resumable-tasks-sycl/sample.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "tbb-resumable-tasks-sycl",
+  "categories": ["Toolkit/Intel® oneAPI Base Toolkit/oneTBB"],
+  "toolchain": ["dpcpp"],
+  "description": "This sample illustrates how computational kernel can be split for execution between CPU and GPU using TBB resumable task and parallel_for. The TBB resumable task uses SYCL to implement calculations on GPU while the parallel_for algorithm does CPU part of calculations. This tbb-resumable-tasks-sycl sample code is implemented using C++ and SYCL language for CPU and GPU.",
+  "languages": [{"cpp":{}}],
+  "targetDevice": ["CPU", "GPU"],
+  "os":["linux", "windows"],
+  "builder": ["ide", "cmake"]
+}
+
```

Comment:
Need to add the ci tests object, for instructions [see the wiki](https://github.com/oneapi-src/oneAPI-samples/wiki/sample-json-ci-test-object)